assets/sass/components/footer.scss

```diff
@@ -0,0 +1,22 @@
+.footer {
+  position: absolute;
+  bottom: 0px;
+  left: 0px;
+  right: 0px;
+  background-color: $color-primary;
+  padding: 20px;
```

Comment:
You can use `2em` or `2rem` here. They are equivalent at this point, and I have not figured out when using `rem`s is better.